manifests/transport_https.pp

```diff
@@ -0,0 +1,8 @@
+# Allow APT to use repos served over HTTPS
+class apt::transport_https (
+  $ensure = 'present'
+) {
+  package { ['apt-transport-https', 'ca-certificates']:
```

Comment:
I started with this `ensure_packages`, but the error message is less clear if you do actually end up with duplicate packages (e.g. if you need to `ensure => 'latest'` and the other package is `ensure => present`).

I also ran into some weird errors with it, but that might be due to our version of puppet (3.4).

I'll give it another go!